### ISSUE TITLE
StargateTech2 spawned gates break my client.

### ISSUE THREAD

**Issue by PixelGamerX32** [StargateMC/StargateMCPublic#372]
There is some stargatetech2 gates that have spawned into the world I am on.
One of them is near my factions current base location.
![gate_location by mace_windu1](https://cloud.githubusercontent.com/assets/21307676/20069049/e2f94926-a512-11e6-85bb-b71c9da73086.png)
This is causing me some problems going near to it, when it attempts a render, it crashes.
I have the log file as well.
Log file details:
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x00007fff51323060, pid=6164, tid=0x00000000000016a0
#
# JRE version: Java(TM) SE Runtime Environment (8.0_101-b13) (build 1.8.0_101-b13)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.101-b13 mixed mode windows-amd64 compressed oops)
# Problematic frame:
# C  [ig7icd64.dll+0x93060]
#
# Failed to write core dump. Minidumps are not enabled by default on client versions of Windows
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
...
Java frames: (J=compiled Java code, j=interpreted, Vv=VM code)
J 19293  org.lwjgl.opengl.GL11.nglCallList(IJ)V (0 bytes) @ 0x0000000005b61ec4 [0x0000000005b61e80+0x44]
J 24134 C2 org.lwjgl.opengl.GL11.glCallList(I)V (19 bytes) @ 0x0000000006f1f818 [0x0000000006f1f7c0+0x58]
j  lordfokas.stargatetech2.transport.RenderStargateTile.renderOuterRing()V+44
j  lordfokas.stargatetech2.transport.RenderStargateTile.render(Lnet/minecraft/tileentity/TileEntity;Lnet/minecraft/block/Block;Lnet/minecraft/world/World;Llordfokas/stargatetech2/core/Vec3Int;F)V+64
j  lordfokas.stargatetech2.core.base.BaseTESR.func_147500_a(Lnet/minecraft/tileentity/TileEntity;DDDF)V+216
j  net.minecraft.client.renderer.tileentity.TileEntityRendererDispatcher.func_147549_a(Lnet/minecraft/tileentity/TileEntity;DDDF)V+22
J 22566 C1 net.minecraft.client.renderer.tileentity.TileEntityRendererDispatcher.func_147544_a(Lnet/minecraft/tileentity/TileEntity;F)V (114 bytes) @ 0x0000000005bf7dc4 [0x0000000005bf7560+0x864]
J 23349 C2 net.minecraft.client.renderer.RenderGlobal.func_147589_a(Lnet/minecraft/entity/EntityLivingBase;Lnet/minecraft/client/renderer/culling/ICamera;F)V (1225 bytes) @ 0x0000000006b462ac [0x0000000006b45120+0x118c]
J 21632 C1 net.minecraft.client.renderer.EntityRenderer.func_78471_a(FJ)V (1918 bytes) @ 0x0000000006560f7c [0x0000000006558600+0x897c]
J 19615 C1 net.minecraft.client.renderer.EntityRenderer.func_78480_b(F)V (1366 bytes) @ 0x0000000005dba044 [0x0000000005db6980+0x36c4]
J 19633 C1 net.minecraft.client.Minecraft.func_71411_J()V (868 bytes) @ 0x0000000005dda5fc [0x0000000005dd7d20+0x28dc]
j  net.minecraft.client.Minecraft.func_99999_d()V+61
j  net.minecraft.client.main.Main.main([Ljava/lang/String;)V+1015
v  ~StubRoutines::call_stub
j  sun.reflect.NativeMethodAccessorImpl.invoke0(Ljava/lang/reflect/Method;Ljava/lang/Object;[Ljava/lang/Object;)Ljava/lang/Object;+0
j  sun.reflect.NativeMethodAccessorImpl.invoke(Ljava/lang/Object;[Ljava/lang/Object;)Ljava/lang/Object;+100
J 3003 C1 sun.reflect.DelegatingMethodAccessorImpl.invoke(Ljava/lang/Object;[Ljava/lang/Object;)Ljava/lang/Object; (10 bytes) @ 0x000000000343cf84 [0x000000000343ce80+0x104]
J 3002 C1 java.lang.reflect.Method.invoke(Ljava/lang/Object;[Ljava/lang/Object;)Ljava/lang/Object; (62 bytes) @ 0x0000000003424cac [0x00000000034248c0+0x3ec]
j  net.minecraft.launchwrapper.Launch.launch([Ljava/lang/String;)V+661
j  net.minecraft.launchwrapper.Launch.main([Ljava/lang/String;)V+8
v  ~StubRoutines::call_stub
...
Whole file is here:
[Log Files.zip](https://github.com/StargateMC/StargateMCPublic/files/576205/Log.Files.zip)
I have multiple files with the same render errors, all pointing to StargateTech2 gates.
Hope this helps to look into.

**Comment by Mace-Windu1** [comment]
Really, in the short term we just need the gate removed, as we have been unable to remove it. We tried explosives, a nuke, and an atomic disassembler. The stargate is on C8L-390, the base blocks go from X: -2678, Z: +133 to X: -2678, Z: +137, the bottom of the gate is at Y:74, with the top at Y:79.

**Comment by Bysokar** [comment]
This issue is a lack of support for old intel graphics cards. Minecraft v1.7.10 (without mods) does not support your computer.
This is the same issue that occurs with rings.

I'll have a look at removing the blocks in question but you'll just have this issue repeat itself if anyone uses a ring block etc.

**Comment by mainesi** [comment]
Lol sokar he knows about the rings etc it's the fact that the gate has spawned when it shouldn't have. Once gone it will not cause any issues

**Comment by Bysokar** [comment]
I'm going to create a client side config which disables animation rendering for players who elect to disable it.

**Comment by PixelGamerX32** [comment]
True. I am using old intel _intergrated_ graphics. I am planning to get a better pc when I can. Until then, StargateTech2 block items are crashing my left, right and centre. Even in Single Player is still having that problem. Must be a old and crappy graphics.
